## Questão 1

Um intervalo de confiança para a proporção populacional pode ser calculado como: $$\big< \bar{p} - z_{1-\alpha/2} \sqrt{\dfrac{\bar{p}(1-\bar{p})}{n}}; \bar{p} + z_{1-\alpha/2} \sqrt{\dfrac{\bar{p}(1-\bar{p})}{n}} \big>,$$ em que $\bar{p}$ é a proporção amostral. Uma amostra aleatória de 400 pessõas apresentou 100 respostas `Sim`. 

- a) Qual é a estimação por ponto da proporção da população que apresentaria respostas `Sim`?.
- b) Calcule um intervalo de confiança para a proporção populacional.
- c) Qual é a margem de erro da sua estimação por ponto?

#### Solução

Lembre-se $\bar{p}$ ou $\hat{p}$ são notações diferentes para representar à proporção estimada.

In [1]:
# a)
p_chapeu = 100/400
p_chapeu

[1] 0.25

In [2]:
# b)
# Como ninguém falou qual o nível de confiança, usamos 95% (ou seja alpha = 5%)
n = 400
alpha = 0.05
z = qnorm(1 - alpha/2)
c(p_chapeu - z * sqrt((p_chapeu * (1 - p_chapeu))/n), p_chapeu + z * sqrt((p_chapeu * (1 - p_chapeu))/n))


[1] 0.2075655 0.2924345

O Intervalo de confiança 95% para a proporção populacional é $$(0.21, 0.29)$$

In [3]:
# c)


Um Intervalo de confiança é da forma $$\text{Estimação pontual} \pm \text{Margem de Erro}$$

In [4]:
# Margem de Erro:
z * sqrt((p_chapeu * (1 - p_chapeu))/n)

[1] 0.04243447

## Questão 2

Uma pesquisa realizada com 346 pessoas da cidade de Rio de Janeiro apontou que 257 delas utilizam máscara de proteção facial quando saem de casa.


- a) Estime a proporção a pessoas na cidade de Rio de Janeiro que usam máscara de proteção facial quando saem de casa.
- b) Qual a margem de erro da sua estimativa?
- c) O ministério da saude está interessado em saber se **pelo menos** 75\% dos moradores do Rio de Janeiro utilizam máscara de proteção facial quando saem de casa. Estabeleça apropriadamente as hipóteses e faça o teste correspondente.


#### Solução


In [5]:
# a)
p_chapeu = 257/346
p_chapeu

[1] 0.7427746

In [6]:
# b)
n = 346
alpha = 0.05
z = qnorm(1 - alpha/2)

# Margem de Erro:
z * sqrt((p_chapeu * (1 - p_chapeu))/n)

[1] 0.04605699

In [7]:
# c)
# H0: p >= 0.75  vs.   H1: p < 0.75

Para $n$ grande, temos: $$Z = \dfrac{\hat{p} - p_0}{\sqrt{p_0 (1 - p_0)/n}} \sim N(0,1)$$


In [8]:
p0 = 0.75
numerador_Z = p_chapeu - p0
denominador_Z = sqrt(p0*(1 - p0)/n)
Z = numerador_Z/denominador_Z
Z

[1] -0.3103854

Como a hipótese é unilateral com $H_1: p < 0.75$, rejeitamos $H_0$ se $Z < z_{\alpha}$

In [9]:
z_alpha = qnorm(alpha)
z_alpha

[1] -1.644854

In [10]:
# -0.310385427622101 <  -1.64485362695147 ? Não, então não rejeitamos H0.

## Questão 3

Uma amostra de 134 professores universitários de rede pública produziu um salário médio de $\bar{x} = 10113.56$ reais com um desvio padrão amostral de $\hat{\sigma}_x = 200.5$ reais. Uma amostra de 245 professores universtirários da rede privada produzio um salário médio de $\bar{y} = 13145.76$ reais com um desvio padrão amostral de $\hat{\sigma}_y = 503.2$. 

- a) Teste se a média salarial dos professores universitários da rede pública é 10250 reais.
- b) Teste se a média salarial dos professores universitários da rede privada é maior que 13000 reais.

#### Solução

Veja que (a) e (b) são perguntas diferentes.

a)

$$H_0: \mu = 10250 \quad vs \quad H_1: \mu \neq 10250$$

em que $\mu$ é a media salarial dos professores universitários da rede pública.

Nota que temos $\hat{\sigma}$ mas não $\sigma$, ou seja, a variância é desconhecida (temos apenas a variância amostral).

In [11]:
mu0 = 10250
n = 134
sigma_chapeu = 200.5


$$T = \dfrac{\bar{X} - \mu_0}{ \hat{\sigma} / \sqrt{n}} \sim t_{n-1}$$

In [12]:
x_barra = 10113.56
numerador_T =  x_barra - mu0
denominador_T = sigma_chapeu/sqrt(n)
numerador_T/denominador_T

[1] -7.877343

Como a hipótese é bilateral, rejeitamos $H_0$ se $|T| > t_{1 - \alpha/2, n-1}$ 

In [13]:
qt(1 - alpha/2, n-1)

[1] 1.977961

|-7.87734257863693| = 7.87734257863693 é maior que 1.97796126416772? **Sim,** então rejeitamos H_0

b) é semelhante.

## Questão 3


A empresa de consultoría `Silva e Associados` deseja fazer um estudo sobre o gasto médio (em produtos básicos) das familias em situação de pobreza no Rio de Janeiro durante a pandemia. Este estudo ajudará a adotar melhores medidas de assistência para combater os efeitos econômicos da pandemia. Um estudo piloto determinou que $\sigma = 55.65$. Determine o tamanho amostral necessário para ter uma margem de erro de 5 reais. Considere $\alpha = 0.05$

#### Solução

Para determinar o tamanho amostral, precisamos utilizar a seguinte formula:  $$n = Z_{1 - \alpha/2}^2 \dfrac{\sigma^2}{E^2}$$

> Lembre-se, sempre arredondamos n pra cima!



In [14]:
alpha = 0.05
Z = qnorm(1 - alpha/2)
sigma = 55.65 # dado no exercício.
E = 5  # E é a margem de erro e também é dado no exercício
n = Z^2 * sigma^2 / E^2
n

[1] 475.868

Precisamos um tamanho amostral de 476.

## Questão 4



O Centro Acadêmico da FACC está interessado em saber se o rendimento acadêmico na matéria MAD211 no 2020.2 foi melhor do que no 2020.1. Para isto, apenas alunos que fizeram MAD211 pela primeira vez foram considerados no estudo. Os resumos amostrais de amostrais de cada uma das pupulações são apresentados a seguir: 

-  **Amostra 1** (alunos que cursaram MAD211 no 2020.1): $n = 23$, $\bar{x} = 7.3$ e $\hat{\sigma} = 0.9$ 
- **Amostra 2** (alunos que cursaram MAD211 no 2020.2): $n = 19$, $\bar{x} = 7.9$ e $\hat{\sigma} = 1.4$ 


Com base na evidência fornecida, poderiamos dizer que o rendimento acadêmico na matéria MAD211 no 2020.2 foi melhor do que no 2020.1?

#### Solução

Estamos em um problema de comparação de médias com variâncias desconhecidas.

- Primeiro precisamos testar se as variâncias desconhecidas são iguais ou diferentes (para, dependendo desse resultado, escolher a estatistica de teste apropriada) $$H_0: \sigma_1 = \sigma_2 \quad vs. \quad H_1:  \sigma_1 \neq \sigma_2$$
- Testemos as médias com variâncias deconhecidas iguais ou diferentes (dependendo do resutado anterior)

> Fizemos uma questão parecida nas duas últimas aulas. Não farei esse gabarito dessa questão.




## Questão 5

Amostras aleatórias doram selecionadas de três populações. Os dados obtidos são os seguintes: 

- **Amostra 1:** 93, 98, 107, 102
- **Amostra 2:** 77, 87, 84, 95, 85, 82
- **Amostra 3:** 88, 75, 73, 84, 74

Com um nível de significância de 5\%, podemos rejeitar a Hipótese nula de que as médias das três populações são iguais?

#### Solução

> Obs: é bem parecido com o exercício final da última aula.



In [15]:
# Preenchemos as observações
amostra1 = c(93, 98, 107, 102)
amostra2 = c(77, 87, 84, 95, 85, 82)
amostra3 = c(88, 75, 73, 84, 74)
amostra_global = c(amostra1, amostra2, amostra3) # todos os elementos

# Calculamos as médias:
m_g = mean(amostra_global)  # média global
m_1 = mean(amostra1) # média da amostra 1
m_2 = mean(amostra2) # média da amostra 2
m_3 = mean(amostra3) # média da amostra 3

# Tamanhos de amostra em cada grupo
n1 = length(amostra1) # Obs na amostra 1
n2 = length(amostra2) # Obs na amostra 2
n3 = length(amostra3) # Obs na amostra 3
nT = n1 + n2 + n3     # Total de Obs (juntando todos os grupos)

In [16]:
# Imprimimos os ns apenas para ver os valores
n1
n2
n3
nT

[1] 4

[1] 6

[1] 5

[1] 15

In [17]:
# Calculamos as somas de quadrados para preencher a tabela

# Soma de Quadrados Totais
SQT = sum((amostra_global - m_g)^2) 
# Soma de Quadrados dos Tratamentos
SQTr = n1*(m_1-m_g)^2 + n2*(m_2-m_g)^2 + n3*(m_3-m_g)^2
# Soma de Quadrados dos Erros
SQE = SQT - SQTr
# Imprimindo resultados
c(SQT, SQTr, SQE)


[1] 1502.933 1036.133  466.800

| Fonte de Variação | g.l  |  Soma dos Q.  | Q. Médios | F     |
|:-----------------|:----|:--------------------:|:-----------------:|:----:|
| Tratamento        |  2 |   1036.813             | 1036.813 /2 | (1036.813 /2 )/(466.8/11) |
| Erro              | 12 |   466.8         | 466.8/12 |        |
| Total             | 14  | `             |                       |        |

In [18]:
(1036.813 /2 )/(466.8/11)

[1] 12.21609

In [19]:
alpha = 0.05
qf(1-alpha, 3 - 1, 15 - 3)

[1] 3.885294

12.2160914738646 > 3.88529383465239? **Sim,** então rejeitamos $H_0$